mothernet/tests/test_follow_sklearn_interface.py

```diff
@@ -83,13 +85,48 @@ def test_additive_mothernet_dense():
     assert classifier.score(X_test, y_test) > 0.9
 
 
+def test_additive_mothernet_dense_logistic_regression():
+    X, y = linear_correlated_logistic_regression(
+        n_features=3, n_tasks=1, n_datapoints=1000, sampling_correlation=0.0)
+    X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=42)
+    model_string = "additive_1_gpu_02_14_2024_16_34_15_epoch_950_fixed2.cpkt"
+    model_path = get_mn_model(model_string)
+    classifier = MotherNetAdditiveClassifier(device='cpu', path=model_path)
+    classifier.fit(X_train, y_train)
+
+    # Plot shape function
+    bin_edges = classifier.bin_edges_
+    w = classifier.w_
+    plot_shape_function(bin_edges, w)
+    print(classifier)
+    prob = classifier.predict_proba(X_test)
+    assert (prob.argmax(axis=1) == classifier.predict(X_test)).all()
+    assert classifier.score(X_test, y_test) > 0.9
+
+
+def test_additive_mothernet_dense_step_function():
```

Comment:
Is this meant as a smoke test? I don't see what it tests, since I think the API is tested in other places already.

Reply:
ok makes sense! These are all run on CI so take time and shouldn't block.